### DISCUSSION TITLE
Expose multiple ports (Truenas)

### DISCUSSION THREAD

**Discussion by migueel15** [itzg/docker-minecraft-server#3013] [Q&A]
Im hosting a local server in TRUENAS. Im using curseforge loader and i have a mod (simple voice chat) that needs a port to be forwarded in udp. The problem is that i can only expose one port via the gui of TRUENAS. I dont know if i can acces the docker-compose file and edit it. Is there some env variable that i can add to expose more than one port?
![image](https://github.com/user-attachments/assets/ba4b1f05-b6f7-4eb0-bd2f-4df5e59d2a24)

**Comment by itzg** [comment]
You'll need to ask the maintainers of that TrueNAS chart. They seem to have imposed the limit. Normally containers can map as many ports as you need, as you mentioned via compose for example.
